docs/public/install.sh

```
@@ -0,0 +1,329 @@
+      fi
+    else
+      printf "${YELLOW}ⓘ Make sure ${BLUE}$INSTALL_PATH${YELLOW} is in your ${BLUE}PATH.${RESET}\n"
+    fi
+    if [ -n "$IS_ROOT" ]; then
+      printf "${YELLOW}ⓘ To synchronize all repos, run: ${GREEN}soar sync --system${RESET}\n"
+    else
+      printf "${YELLOW}ⓘ To synchronize all repos, run: ${GREEN}soar sync${RESET}\n"
+    fi
+    # Check Current Config
+    if [ -n "$IS_ROOT" ]; then
+      SOAR_ENV_OUT="$($INSTALL_PATH/soar env --system 2>/dev/null)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)